platform/ios/src/MGLAnnotationView.h

```diff
@@ -169,12 +169,12 @@ MGL_EXPORT
  value of this property is `NO` or the map’s pitch is zero, the annotation view
  remains the same size regardless of its position on-screen.
 
- The default value of this property is `YES`. Set this property to `NO` if the
- view’s legibility is important.
+ The default value of this property is `NO`. Keep this property set to `NO` if
+ the view’s legibility is important.
```

Comment:
Per #5040, https://github.com/mapbox/mapbox-gl-js/issues/4120#issuecomment-300933275, and mapbox/mapbox-gl-js#4547, we don’t allow developers to turn off scaling GL point annotations or symbol layers based on viewing distance:

> …or if the map also represents some annotations with `MGLAnnotationImage` objects, or if the style also contains symbol layers that the user can interact with.

Reply:
I'm not familiar enough with how `MGLAnnotationView` is typically used to have a strong opinion here. But if I'm reading this all correctly, isn't it the case that either way, the scaling for annotation views will be 50% off from symbol layers (because they use the hardwired 50% "attenuated" pitch-scaling, and this functionality is either 0 or 100%)?

Reply:
I’m not sure if the pitch-scaling for view-backed annotations would be 100% – it’s a bit of a hack because we don’t have access to the map transform at this level (see also #10498). So I guess there’s inconsistency regardless. 👍